WordPressKitTests/SelfHostedPluginManagementClientTests.swift

```diff
@@ -0,0 +1,108 @@
+import Foundation
+import XCTest
+import OHHTTPStubs
+
+@testable import WordPressKit
+
+class SelfHostedPluginManagementClientTests: XCTestCase {
+
+    override func setUp() {
+        super.setUp()
+
+        stub(condition: { _ in true }) {
+            XCTFail("Unexpected request: \($0)")
+            return HTTPStubsResponse(error: URLError(URLError.Code.networkConnectionLost))
+        }
+    }
+
```

Comment:
I believe we should also remove the stubs?

```suggestion
    override func tearDown() {
        super.tearDown()
        HTTPStubs.removeAllStubs()
    }
```

Reply:
Good catch! I always forget about it. Addressed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/724/commits/c44bb13e7c5551471274efe47ee59f1374f6f9e4.